mac/build.sh

```diff
@@ -290,12 +299,6 @@ execBuildCommand() {
     eval $cmnd
     ret_code=$?
     set -e
-    if [ $ret_code == 65 ]; then
```

Comment:
Now that code signing is in its own block, we don't ever need to retry the build step